heron/spi/tests/java/com/twitter/heron/spi/utils/SchedulerUtilsTest.java

```diff
@@ -168,4 +175,63 @@ public void testSchedulerCommandArgs() throws Exception {
     Assert.assertArrayEquals(expectedArgs, SchedulerUtils.schedulerCommandArgs(
         Mockito.mock(Config.class), Mockito.mock(Config.class), freePorts));
   }
+
+  @Test
+  public void updatesPackingPlanWithUniformSizedContainers() {
+    SchedulerStateManagerAdaptor adaptor = Mockito.mock(SchedulerStateManagerAdaptor.class);
+    Mockito.when(adaptor.deletePackingPlan("topology")).thenReturn(true);
+    Mockito.when(adaptor.setPackingPlan(Mockito.any(PackingPlans.PackingPlan.class),
+        Mockito.eq("topology"))).thenReturn(true);
+    Config runtime = Mockito.mock(Config.class);
+    Mockito.when(runtime.get(Keys.schedulerStateManagerAdaptor())).thenReturn(adaptor);
+
+    PackingPlan packing = createPackingPlan();
+    SchedulerUtils.persistUpdatedPackingPlan("topology", packing, runtime);
+    InOrder inOrder = Mockito.inOrder(adaptor);
+    inOrder.verify(adaptor).deletePackingPlan("topology");
+    inOrder.verify(adaptor).setPackingPlan(Mockito.any(PackingPlans.PackingPlan.class),
+        Mockito.eq("topology"));
+  }
+
+  @Test
+  public void updatesPackingPlanInStateManager() {
```

Comment:
Ok. A public method can have more than one test method to test different control paths. So I plan to follow the following rules:

1. Include some test information in the name. That way if the test fails, a hint will be available to the developer. 
1. To avoid making the test name too long avoid redundant word `test`. The method belongs to a test class and is `Test` annotated. When the test fails, the word test does not add any value, just makes the method name long.